web/content/docs/benchmarks/heatconduction/heatconduction-line_source_term.pandoc

```diff
@@ -0,0 +1,118 @@
++++
+author = "Thomas Fischer"
+date = "2019-10-01T11:54:02+01:00"
+title = "Heatconduction (Line Source Term)"
+weight = 121
+project = "Parabolic/T/1D_line_source_term_tests/line_source_term.prj"
+
+[menu]
+  [menu.benchmarks]
+    parent = "heatconduction"
+
++++
+
+{{< data-link >}}
+
+## Equations
+
+We consider the Poisson equation:
+$$
+\begin{equation}
+\nabla\cdot(\nabla T) + Q_T = 0 \quad \text{in }\Omega
+\end{equation}$$
+w.r.t Dirichlet-type boundary conditions
+$$
+\eqalign{
+T(x) = 0 &\quad \text{on }\Gamma_D,\cr
+}
+$$
+where $T$ could be temperature, the subscripts $D$ denotes the Dirichlet-type
+boundary conditions. Here, the temperature distribution under the impact of a
+line shaped source term should be studied.
+
+## Problem Specifications and Analytical Solution
+
+In OGS there are several benchmarks for line source terms in 2d and 3d domains
+available. Here, some of the 3d benchmarks are described.
+
+### Cylindrical domain
+
+The Poisson equation on cylindrical domain of heigth $1$ and radius
+$r=1$ is solved. In the following figure the geometry, partly semi-transparent,
+is sketched. Furthermore, the mesh resolution is shown in the cylindrical domain
+within the first quadrant of the coordinate system. In the second quadrant the
+simulated temperature distribution is depicted.
+
+{{< img src="../LineSourceTermFigures/temperature_distribution_line_source_term_in_cylinder.png" >}}
+
+The source term is defined along the line in the center of the cylinder:
+$$
+\begin{equation}
+Q(x) = 1 \quad \text{at } x=0, y=0.
+\end{equation}
+$$
+In the above figure the source term is the red vertical line in the origin of
+the coordinate system.
+
+The analytical solution for a line source in the cylinder is
+$$
+\begin{equation}
+T(x) = - \frac{1}{2 \pi} \ln \sqrt{x^2 + y^2}.
+\end{equation}
+$$
+
+#### Analytical solution in paraview
+
+Since the analytical solution has a singularity at $(x, y) = (0, 0)$ the
+analytical solution in paraview is generated as follow:
+```
+if (coordsX^2<0.0001 & coordsY^2<0.0001, temperature, -1/(4*asin(1))*ln(sqrt(coordsX^2+coordsY^2))
+```
+
+#### Results and evaluation
+
+- Comparison with analytical solution:
```

Comment:
One recommendation: Here the temperature profile curve should be plotted at least once. Normal readers may not be familiar with the problem. They maybe confused as the error plot was directly shown.

Reply:
Thanks for the recommendation. I added the temperature profile.